cmd/estuary-shuttle/main.go

```diff
@@ -477,6 +477,11 @@ func main() {
 			}
 		})
 
+		debounceCache, err := lru.New(2000)
```

Comment:
Is there a reason why we use ‘debounceCache’ instead of ‘eventCache’?

2000 should be a configuration somewhere but set a default const to 2000. We can name it to debounceCacheSize.

@gmelodie the name ‘debounceCache’ make sense since the line initializes the LRU.

Reply:
Humm.. I was thinking that it does not really says _what_ it debounces, but I'm fine with either

Reply:
@alvin-reyes @gmelodie I have addressed all issues